osx32/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+CompilerOptions="-DDFRE_OSX32 "
+CompilerOptions+="-nostdlib "
```

Comment:
Does it not build in this first commit because -nostdlib is there?

Reply:
Not too worried about the not building every commit thing now. You can check and say just for posterity or just resolve this.

Reply:
don't build from this commit peeps ✌️